docs/features/vision/fog_of_war_manager.md

```
@@ -0,0 +1,433 @@
+		return find_child("ScreenOverlay").material_override.get_shader_parameter("debug_texture_view")
+## Shows small texture of the fog
+@export_range(0, 1) var debug_texture_view_size : float = 0.2:
+	set(value):
+		find_child("ScreenOverlay").material_override.set_shader_parameter("debug_texture_view_size", value)
+	get:
+		return find_child("ScreenOverlay").material_override.get_shader_parameter("debug_texture_view_size")
+
+@export_group("Editor Only Circle")
+## TODO: Add description
+@export var editor_only_circle_color : Color = Color.WHITE :
+	set(value):
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)